### ISSUE TITLE
Bug/Enhancement: Cannot upload content / Add removable-media connection [snap only]

### ISSUE THREAD

**Issue by keunes** [ferdium/ferdium-app#202]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.0.0-nightly.49

### What Operating System are you using?

Ubuntu

### Operating System Version

18.04.6 LTS

### What arch are you using?

x64

### Expected Behavior

Allow to upload files on GitHub in Ferdi from my mounted files drive using drag-and-drop.

### Actual Behavior

When dragging-dropping, GitHub attempts to upload a file but says something like 'empty file, please try again'.

### Steps to reproduce

1. Install & open Ferdium on a system with dual boot, where the Windows filesystem is mounted in Ubuntu.
2. Open the GitHub service & start creating a new issue.
3. Drag-and-drop a file into a text field.
4. See error.

### Additional information

Via the [installation page](https://ferdium.org/download) the connections `camera` and `pulseaudio` are added. I believe adding the [`removable-media`](https://snapcraft.io/docs/removable-media-interface) connection would solve my problem - allowing Ferdium to access my Windows filesystem from Ubuntu. (At least - that's a connection I have seem to have set for 0AD, in order to install plug-ins stored on the Windows filesystem.)

When trying `snap connect ferdium:removable-media` I get `error: snap "ferdium" has no plug named "removable-media"`. I also cannot set it via the snap permission settings:
![Ferdium snap permissions](https://user-images.githubusercontent.com/11229646/171109921-60956206-c848-45f0-b16a-70e8a9455fd4.png)

**Comment by NathanaelGandhi** [comment]
This can easily be added similar to #215. I'm happy to do that once that PR is merged if you don't want to add it yourself.

~~The question remaining is should removable-media plug be included in #216?
From a security perspective, I would say no, from a user experience perspective, I would say yes.~~

EDIT: if we add the interface the connection should be done manually by users. User prompt could be added in #216

**Comment by NathanaelGandhi** [comment]
@keunes Once the above PR has been merged can you please test your usecase using ```snap connect ferdium:removable-media``` and report back?

I don't have any windows volumes to mount but I will test mounting external USB devices.

**Comment by SpecialAro** [comment]
Is this fixed?

**Comment by NathanaelGandhi** [comment]
making the manual connection works for me.

@keunes Can you confirm your issue has been fixed using the lastest version and ```snap connect ferdium:removable-media``` ?

**Comment by keunes** [comment]
```
Version: 6.0.0-nightly.96
Server: No Server
Electron: 19.0.8
Chrome: 102.0.5005.148
Node.js: 16.14.2
Platform: Linux 4.15
Arch: x64
Build date: Mon Jul 11 2022 12:22:51 GMT+0200 (Central European Summer Time)
Git SHA: 1342ace5
Git branch: nightly
```

I'm still getting the 'This file is empty.' warning unfortunately when dragging and dropping. Also when clicking on 'browse' I can see the 'Documents' folder which mounts the Windows folder to /files in Ubuntu, but the folder appears empty.

Copying from the clipboard (e.g. this screenshot I took) does work:
![image](https://user-images.githubusercontent.com/11229646/178345489-caaed016-747f-4fbd-a38b-5560f4f35135.png)

I guess I was wrong about the 'removable-media' connector :/

**Comment by keunes** [comment]
I just realised there's another effect of this: I cannot pick custom icons for my tabs (particularly the 'custom website' recipe) :(

Anything I can do to help debug this?

**Comment by keunes** [comment]
@NathanaelGandhi If it's of any help, I just realised that when I open a file dialog (to save/load files), it opens the mapped folders of the snap. So
* Ferdium sees: `/home/keunes/snap/ferdium/97/Downloads`
* Which is a link to: `/home/keunes/Downloads`
* However on my system I automount the Windows drive and made its subfolders the special folders in Ubuntu ([HowTo guide](https://www.howtogeek.com/howto/35807/how-to-harmonize-your-dual-boot-setup-for-windows-and-ubuntu/))
So Ferdium should be looking at `/files/Downloads` instead.

Browsing the filesystem from the Ferdium dialog, however, I see no trace of the `/files` mount. My fstab contains this:
```
# attach data partition
UUID=586CEA196CE9F224	/files	ntfs-3g	auto,user,rw,windows_names,locale=nl_NL.utf8	0	0
```

And I just discovered [this AskUbuntu answer](https://askubuntu.com/a/964403):
> Perhaps it even uses the `removable-media` interface, but that only covers stuff mounted into `/media` or `/mnt`. If your other partitions are mounted elsewhere, the snap will not have permission to follow the symlinks.

:(

**Comment by NathanaelGandhi** [comment]
@keunes have you tried mounting/ mapping /  symlinking your windows dir to a folder snap has access to? Did you find a solution to your problem?

**Comment by keunes** [comment]
Hi
Yes, I mounted it under `/mnt` and (thanks to the change made to Ferdium earlier) I can now access the files. Navigating to files is still a bit of a pain because my special user folders (like 'Documents') aren't shown in the file selection dialog. But I can live with it, and will probably improve when #200 gets addressed.
